indico/modules/events/settings.py

```diff
@@ -261,3 +262,9 @@ def get_themes_for(self, event_type):
     'emails': [],
     'phones': []
 })
+
+event_contribution_settings = EventSettingsProxy('contrib', {
```

Comment:
this should be in `indico/modules/contributions/__init__.py` (just like we do for other modules such as `indico/modules/sessions/__init__.py`)